src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
+}
```

Comment:
The `@font-face` rule is not necessary here because you have already loaded the font files in your project. If you want to use other font weights or styles, you can just specify them in your CSS rules using the `font-weight` and `font-style` properties.